frontend/src/features/settings/types/CreditAccountActivityColumns.tsx

```diff
@@ -0,0 +1,66 @@
+import { MRT_ColumnDef, MRT_Row } from "material-react-table";
+import {
+  CreditAccountActivity,
+  CreditAccountActivityDisplayValues,
+} from "./creditAccount";
+
+const formatDate = (dateString: string): string => {
```

Comment:
Why do you need another formatDate method? Does the current ones under "src/common/helpers/formatDate.ts" not accommodate the use cases/needs?